### PR TITLE
Minor edits to authorship + New record

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: new/LIT5810HomilySatan.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5810HomilySatan" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Homily by Ephrem on Satan and on the death</title>
```
Comment (thea-m) [inline new/LIT5810HomilySatan.xml]: en :)
"and on death" without the?

Comment (eu-genia) [reply new/LIT5810HomilySatan.xml]: corrected (copypasted from E-S but you are right)

Comment (thea-m) [reply new/LIT5810HomilySatan.xml]: sorry, my comment was maybe too elliptic. The title given here is English but marked up as geez.

Comment (eu-genia) [reply new/LIT5810HomilySatan.xml]: ah:)

**Pull request merged**